package.json

```diff
@@ -285,6 +286,7 @@
     "terser-webpack-plugin": "^4.2.0",
     "wait-for-expect": "^3.0.2",
     "webpack": "4.44.1",
+    "webpack-bundle-analyzer": "^4.4.2",
```

Comment:
this is nice to have and i think we should maintain it here, can you add a yarn script for ease of use please ? or maybe document how to use it in the README ?